contrib/cirrus/runner.sh

```diff
@@ -35,9 +35,15 @@ EOF
 }
 
 _run_validate() {
+    rustup install stable
+    rustup default stable
```

Comment:
`rustup` is non-standard in the Fedora distribution of rust.  I don't think the `setup.sh` code-path leading to this function includes the upstream-rust install due to `"$CIRRUS_TASK_NAME" != "build_cross"`

Reply:
Huh, I really don't understand how this is working.  Maybe I'm wrong and Fedora does include `rustup` now?

Reply:
@cevich Correct `fedora` does not ships rust with `rustup` I dunno why ? Because of it our toolchain is really old but its ok i guess. We can do it by ourselves :)